src/app/doc-pages/table-viewer-default-doc/table-viewer-default-doc.component.ts

```diff
@@ -73,6 +86,7 @@ setAll() {
     { apiInput: '[tableConfig]', description: 'The config for the table' },
     { apiInput: '[tableConfigStatic]', description: 'The static version of the config for the table' },
     { apiInput: '[stickyHeader]', description: 'If header should be sticky' },
+    { apiInput: '[paginationOptions]', description: 'An array of the possible elements per page. One for everything will also be added automatically' },
```

Comment:
Litt forvirrende setning. Kanskje: If pagination is wanted, give an array of page-size options that should be available to the user.  The page-size option to see everything in the table will be added by automatically.

Reply:
La inn din med noen endringer